File: class/override.java
class a {
    // int i = 100;
    public void show() {
        System.out.println("100 Parent");
    }
}

class b extends a {
    // int i = 555;
    public void show() {
        System.out.println("555 Child");
    }
}

class override {
    public static void main(String args[]) {
        a obj = new b();
        // System.out.println(obj.i);
        obj.show();
    }
}

File: class/override.cs
class a{       //  csc to compile
    public void show(){
        public int i=100;
    System.Console.WriteLine("Parent");
    }
}
class b : a{
    public void show(){

        public int i=200;
    System.Console.WriteLine("Child");
    }
}
class override{
    public static void Main(){
        b obj=new b();
    System.Console.WriteLine(obj.i);
    }
}